docs/xbox.md

```diff
@@ -0,0 +1,29 @@
+---
+section: legacy
+hide:
+  - next
+  - prev
+  - search
+  - tabs
+---
+
+## Input Plugin
+
+The recommended controller plugin for XBox and other USB controllers is N-Rage V2 1.83, which is included with the emulator. Make sure to select it as your input plugin by going to `Options -> Settings -> Plugins.` These instructions **won't work** if you're using a different input plugin.
+## Configuring Inputs
+
+#### Windows 7 & 8
```

Comment:
Windows 8.1 is big enough to mention, I don't know where it goes. I also think we should probably mention that all of these OS are now EOL. (8.1 goes EOL tomorrow - Jan 10, 2023).

Reply:
Added EOL warning in https://github.com/smash64-dev/smash64-online/pull/10/commits/f93c24ee6dd89005818eb97d9a97a58d4f2a76e2. I don't have 8.1 to test. Probably the same?